MAUI/TabView/Getting-Started.md

```diff
@@ -460,3 +460,4 @@ By defining the `ContentItemTemplate` of the `SfTabView`, a custom user interfac
 
 ![Item Template](images/ItemTemplate.png)
 
+N> You can also explore our [.NET MAUI DataGrid Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/TabView) that shows you how to render the Tab View in .NET MAUI.
```

Comment:
Update as below
N> You can refer to our [.NET MAUI Tab View](https://www.syncfusion.com/maui-controls/maui-tab-view) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Tab View Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/TabView) that shows you how to render the Tab View in .NET MAUI.

Reply:
resolved